client/src/components/formHelpers/form.tsx

```diff
@@ -103,7 +103,6 @@ export const StrictSolutionForm = ({
           <FormFields formFields={formFields} options={options} />
           <BlockSaveButton
             disabled={(pristine && !enableSubmit) || (error as boolean)}
```

Comment:
it worked for me without deleting the prop? why do we need the prop deletion?

```suggestion
            disabled={(pristine && !enableSubmit) || (error as boolean)}
            bgSize='none'
```